### PR TITLE
#1323- Remove Email download from core 

### PR DESCRIPTION
**Pull request by yuntianhu**
Harvard no longer does email downloads, so the special functionality we built in for Harvard is no longer needed

Expected behavior
We just don't want this code anymore, so this will make that go away.

### CODE REVIEW

File: app/javascript/geoblacklight/downloaders/downloader.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+export default class GeoBlackLightDownloader {
+    constructor(el, options) {
+        this.el = document.querySelector(el);
```
Comment (jcoyne) [inline app/javascript/geoblacklight/downloaders/downloader.js:3]: Why is this necessary? It looks like you're passing in an element, not a selector.

File: app/javascript/geoblacklight/downloaders/downloader.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+export default class GeoBlackLightDownloader {
```
Comment (jcoyne) [inline app/javascript/geoblacklight/downloaders/downloader.js:1]: GeoBlacklight is the preferred capitalization.

File: app/javascript/geoblacklight/downloaders/downloader.js (hunk 1 above)
Comment (thatbudakguy) [reply app/javascript/geoblacklight/downloaders/downloader.js:3]: This file can be deleted completely; downloads are handled via the `DownloadController` stimulus controller now.

Comment (yuntianhu) [reply app/javascript/geoblacklight/downloaders/downloader.js:3]: seems like we don't need app/javascript/geoblacklight/downloader.js, also?  I will rebase.

Comment (thatbudakguy) [reply app/javascript/geoblacklight/downloaders/downloader.js:3]: yep, that's right. rebase should fix it

**Review by eliotjordan — APPROVED** [review]
